src/request.ts

```diff
@@ -165,7 +173,7 @@ class Request {
             if (options.headers['x-anonymous']) {
               delete options.headers['x-anonymous'];
             }
-            else {
+            else if(accessToken !== '') {
```

Comment:
We can drop this `else` and use just `if`. Also let's format the code to make it more readable by adding a space between `if` and `(`

Reply:
🤙